eravm-stable-interface/src/tracer_interface.rs

```diff
@@ -208,6 +208,19 @@ impl<T: opcodes::TypeLevelReturnType> OpcodeType for opcodes::Ret<T> {
 pub trait Tracer {
     fn before_instruction<OP: OpcodeType, S: StateInterface>(&mut self, _state: &mut S) {}
     fn after_instruction<OP: OpcodeType, S: StateInterface>(&mut self, _state: &mut S) {}
+
+    fn on_extra_prover_cycles(&mut self, _stats: CycleStats) {}
```

Comment:
Have you checked whether implementations are inlined as expected (i.e., there's no performance regression compared to the previous inlined stats implementation)?

Reply:
I'm assuming there isn't, as at least the integration branch iai slightly improved.